tools/harness-thci/OpenThread.py

```diff
@@ -3186,7 +3222,7 @@ def setCSLperiod(self, period=500):
 
     @staticmethod
     def getForwardSeriesFlagsFromHexStr(flags):
-        hexFlags = int(flags, 16)
+        hexFlags = int(flags, 16) if isinstance(flags, str) else flags
```

Comment:
Probably we can rename this method since `flags` is allowed to be a hex number now.

Reply:
Done.